src/components/toolbar/Toolbar.tsx

```diff
@@ -56,6 +62,7 @@ export interface ToolbarItemProps extends ToolbarBaseProps {
 
 export interface ToolbarPopoverItemProps extends PopoverProps {
   itemProps: ToolbarItemProps;
+  popoverInteraction?: PopoverInteractionType;
```

Comment:
You did not need to invent a new prop to add the behaviour to the item. Popover props are already spread into the `Popover` component so you could have used the original prop for this.

However, it makes sense to pass this option to the `Toolbar` component via a `popoverInteractionKind` prop to avoid having to pass this to every item.

Reply:
I added the `popoverInteractionKind` prop to the toolbar context